guides/source/configuring.md

```diff
@@ -3563,12 +3563,11 @@ development:
   pool: 5
 ```
 
-By default Active Record uses database features like prepared statements and advisory locks. You might need to disable those features if you're using an external connection pooler like PgBouncer:
+By default Active Record a database features called advisory locks. You might need to disable this feature if you're using an external connection pooler like PgBouncer:
```

Comment:
```suggestion
By default Active Record uses a database feature called advisory locks. You might need to disable this feature if you're using an external connection pooler like PgBouncer:
```

Reply:
Thank you, @Earlopain!  Fixed in https://github.com/rails/rails/commit/6a849e3c7e43a0cb19d5b277d0817ca1ad4f4c6b.